configure.ac

```diff
@@ -2502,14 +2502,14 @@ if test "x$enable_python" = "xyes" || \
     AC_MSG_ERROR([only specify one of PYTHON or PYTHON_VERSION])
   fi
 
-  if test -n "$PYTHON"; then
-    PYTHON_VERSION=`$PYTHON -c "import sys; sys.stdout.write(sys.version[[:3]])"`
-  fi
-
   if test -n "$PYTHON_VERSION"; then
     AC_SUBST([PYTHON], [python$PYTHON_VERSION])
   fi
 
+  if test -n "$PYTHON"; then
+    PYTHON_VERSION=`$PYTHON -c "import sys; sys.stdout.write(sys.version[[:3]])"`
+  fi
+
```

Comment:
now we're overriding PYTHON_VERSION if it's already set? (since now we're setting PYTHON above when PYTHON_VERSION is set?)

Reply:
Converted it into an "else if".